tempest/api/workloadmgr/network/test_network_restore.py

```diff
@@ -262,11 +264,129 @@ def test_4_network_restore_cli(self):
         finally:
             reporting.test_case_to_write()
 
-    def test_5_cleanup(self):
+    @decorators.attr(type='workloadmgr_cli')
+    def test_5_network_restore_cli_workload_reassign(self):
+        try:
+            reporting.add_test_script(str(__name__) + "_network_restore_cli_workload_reassign_full_snapshot_cli")
+            tenant_id = CONF.identity.tenant_id
+            tenant_id_1 = CONF.identity.tenant_id_1
+            user_id = CONF.identity.user_id
+
+            rc = self.workload_reassign(tenant_id_1, workload_id, user_id)
+            if rc == 0:
+                LOG.debug("Workload reassign from tenant 1 to tenant 2 passed")
+                reporting.add_test_step(
+                    "Workload reassign from tenant 1 to 2", tvaultconf.PASS)
+            else:
+                LOG.error("Workload reassign from tenant 1 to 2 failed")
+                reporting.add_test_step(

```

Comment:
Set test script status to FAIL in else part. 
Do we need to raise exception and exit in case if reassignment is not happening from tenant1 to tenant2 ? If we are raising exception then we don't need to set test script status to FAIL. It is covered in Exception part.
[[[ reporting.set_test_script_status(tvaultconf.FAIL)
raise Exception("") ]]]